### ISSUE TITLE
add questions to web development quizes and show results

### ISSUE THREAD

**Issue by KapuluruBhuvaneswariVspdbct** [notsoocool/codecache#163]
**Is your feature request related to a problem? Please describe.**
add questions to web development quizes and show results
**Describe the solution you'd like**
add questions to web development quizes and show results
**Describe alternatives you've considered**
add questions to web development quizes and show results
**Additional context**

**Comment by notsoocool** [comment]
@KapuluruBhuvaneswariVspdbct Just a reminder or warning: when you get assigned an issue, kindly complete it to the fullest extent. If you’re only raising PRs for points, please don’t. If I see this kind of behavior again, I will take the necessary actions. 

When you were already assigned to your issue, why didn't you complete it already? Now, you're raising another issue.

**Comment by KapuluruBhuvaneswariVspdbct** [comment]
the changes are not at all displayed in website even after merging that is why i raised the pr

**Comment by notsoocool** [comment]
> the changes are not at all displayed in website even after merging that is why i raised the pr

@KapuluruBhuvaneswariVspdbct its because its in different branch, it will be merged to the deployment branch once it is approved by all the team members, I merged you PR so that atleast you can get points for your work.

**Comment by KapuluruBhuvaneswariVspdbct** [comment]
not about the points sir,i just want you to look at the sample quiz conducted  based on your review ,if you want me to proceed i would have added all other quizes in a single pr sir

**Comment by KapuluruBhuvaneswariVspdbct** [comment]
thanlk you sir

**Comment by notsoocool** [comment]
@KapuluruBhuvaneswariVspdbct Could you please work on a different issue? I don’t want to continue with the quiz feature for now.

**Comment by KapuluruBhuvaneswariVspdbct** [comment]
Actually I have added the quizes the incomplete work is adding questions to
remaining issues

**Comment by notsoocool** [comment]
That’s the whole point I was referring to earlier: when an issue is assigned to you, it’s important to complete it to your fullest extent and not just raise a partially complete PR.

**Comment by notsoocool** [comment]
Because of this I am not going to entertain this issue any further

**Comment by KapuluruBhuvaneswariVspdbct** [comment]
I have added the quiz cards and added the different topic cards and a
sample quiz if it is reviewed by you then I can add all the quizes ,if is
not about creating quizes ,I wanted to add the questions to all quizes
based on your review

**Comment by KapuluruBhuvaneswariVspdbct** [comment]
Ok sir I will respect your decision,it is up to you to assign or close the
issue sir
